delivery_gefco/README.rst

```diff
@@ -0,0 +1,115 @@
+==============
+Delivery Gefco
+==============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
+    :target: https://github.com/OCA/delivery-carrier/tree/13.0/delivery_gefco
+    :alt: OCA/delivery-carrier
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/delivery-carrier-13-0/delivery-carrier-13-0-delivery_gefco
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/99/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module adds `Gefco <https://gefco.net>`_ to the available carriers.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+This module depends on the OCA/delivery-carrier`delivery_package_number` module.
+
+The Gefco service doesn't provide delivery rating methods, so OCA's
+`delivery_price_method` is advised in order to use this carrier in a sales workflow.
+
+Configuration
+=============
+
+To configure a Gefco delivery method:
+
+#. Go to *Inventory > Configuration > Delivery > Shipping methods* and create a new one.
+#. Choose *Gefco* as provider.
+#. Configure *Gefco Agency Code* and "Gefco Shipper Identification" fields.
+#. Go to *Inventory > Configuration > Delivery > Gefco Destinations* and import records.
+
+IMPORTANT: It is necessary to import the Gefco destinations that are needed according 
+to the shipping countries of the partners. /data/gefco.destination.csv
```

Comment:
Why not to load them directly on the manifest?

Reply:
This is a performance issue because the file has almost 150,000 records.

Reply:
Mm, doesn't the orm create them in one go? Did you test it?

Reply:
Anyways that file seems to just reproduce the Spanish ZIP codes and must disappear for using standard data instead.

Reply:
Gefco has specific codes for each postal code in each country that must be specified on the label.
Do you think it would be better to use the table from `res.better.zip` instead? (the problem of updating that number of records may be the same).